internal/git/core.go

```diff
@@ -171,20 +145,24 @@ func (cache *gitCache) cloneRepo(repo string, commit string, branch string) chan
 		cloneCommand.Env = append(cloneCommand.Env, os.Environ()...)         // pass all env variables to git command so proper SSH config is passed if needed
 		cloneCommand.Env = append(cloneCommand.Env, "GIT_TERMINAL_PROMPT=0") // tell git to fail if it asks for credentials
 
-		if output, err := cloneCommand.CombinedOutput(); err != nil {
-			logger.Error(emoji.Sprintf(":no_entry_sign: Error occurred while cloning: '%s'\n%s: %s", cacheToken, err, output))
-			cloneResultChan <- &gitCloneResult{Error: err}
+		var stdout, stderr bytes.Buffer
+		cloneCommand.Stdout = &stdout
+		cloneCommand.Stderr = &stderr
+		if err := cloneCommand.Run(); err != nil {
+			cloneResultChan <- &gitCloneResult{Error: fmt.Errorf("%v: %v", err, stderr.String())}
 			return
 		}
 
 		// If commit provided, checkout the commit
 		if len(commit) != 0 {
 			logger.Info(emoji.Sprintf(":helicopter: Performing checkout commit '%s' for repo '%s'", commit, repo))
 			checkoutCommit := exec.Command("git", "checkout", commit)
+			var stdout, stderr bytes.Buffer
+			checkoutCommit.Stdout = &stdout
+			checkoutCommit.Stderr = &stderr
 			checkoutCommit.Dir = clonePathOnFS
-			if output, err := checkoutCommit.CombinedOutput(); err != nil {
-				logger.Error(emoji.Sprintf(":no_entry_sign: Error occurred checking out commit '%s' from repo '%s'\n%s: %s", commit, repo, err, output))
```

Comment:
What was the reasoning to remove this logging line?

Reply:
woops, forgot to re-add that. re-addded.